tests/test_admin_and_files.py

```diff
@@ -27,7 +27,7 @@ def test_commentaries_registered_in_admin(self):
 
 class GitignoreTests(TestCase):
     def test_gitignore_exist(self):
-        file_exists = os.path.exists('.gitignore')
+        file_exists = os.path.exists(".gitignore")
         assert file_exists
```

Comment:
The `test_gitignore_exist` method uses a plain `assert` statement. It's recommended to use `self.assertTrue(file_exists)` for consistency with Django's `TestCase` assertions.